browser/profiles/brave_renderer_updater.cc

```diff
@@ -191,7 +192,11 @@ void BraveRendererUpdater::UpdateRenderer(
 
   PrefService* pref_service = profile_->GetPrefs();
   bool de_amp_enabled = de_amp::IsDeAmpEnabled(pref_service);
-  bool widevine_enabled = local_state_->GetBoolean(kWidevineEnabled);
+  bool widevine_enabled = false;
+  // Remove when we have support of Widevine on Linux arm64
+#if !BUILDFLAG(IS_LINUX) || !defined(ARCH_CPU_ARM64)
+  widevine_enabled = local_state_->GetBoolean(kWidevineEnabled);
```

Comment:
I believe the correct fix is to put the declaration of `kWidevineEnabled` under the build flag. It gives us compile-time checking.

Otherwise we could miss the things like that: https://github.com/brave/brave-core/blob/master/browser/profiles/brave_renderer_updater.cc#L69C1-L69C1

Reply:
yep, you are right, added `#if BUILDFLAG(ENABLE_WIDEVINE)`